src/chroot_script

```diff
@@ -212,6 +212,25 @@ echo "mapping wlan1" >> /etc/network/interfaces
 echo "  script /root/bin/map_iface" >> /etc/network/interfaces
 echo "source /boot/octopi-network.txt" >> /etc/network/interfaces
 
+# allow wifi network configuration via /boot/octopi-wifi.txt
+mv /etc/wpa_supplicant/wpa_supplicant.conf /boot/octopi-wifi.txt
+ln -s /boot/octopi-wifi.txt /etc/wpa_supplicant/wpa_supplicant.conf
```

Comment:
I'd suggest calling it something other than `octopi-wifi.txt` - too similar to `octopi-network.txt` in the mind of people ("something with the network it was... ah, wifi!"). Why not `wpa_supplicant.conf`? Or better yet, a new subdirectory `advanced` or something and putting it in there? Plus adding a big disclaimer comment at the top "This is only used for advanced wifi configuration, take a look into octopi-network.txt instead if you only need to configured one singular wifi network", and a hint in octopi-network.txt along the lines of "If you need to configure more than one wifi network and be able to roam between them, please leave everything in here commented and add your credentials to advanced/wpa_supplicant.conf instead".